index.html

```diff
@@ -49,579 +49,640 @@
-			if (contract.assembly !== null)
-			{
-				details.append($('<span class="col1">Assembly</span>'));
-				var assembly = $('<pre/>').text(formatAssemblyText(contract.assembly, '', source));
-				details.append(assembly);
-			}
-			button.click(function() { detailsOpen[contractName] = !detailsOpen[contractName]; details.toggle(); });
-			if (detailsOpen[contractName])
-				details.show();
-			return $('<div class="contractDetails"/>').append(button).append(details);
-		};
-		var formatGasEstimates = function(data) {
-			var gasToText = function(g) { return g === null ? 'unknown' : g; }
-			var text = '';
-			if ('creation' in data)
-				text += 'Creation: ' + gasToText(data.creation[0]) + ' + ' + gasToText(data.creation[1]) + '\n';
-			text += 'External:\n';
-			for (var fun in data.external)
-				text += '  ' + fun + ': ' + gasToText(data.external[fun]) + '\n';
-			text += 'Internal:\n';
-			for (var fun in data.internal)
-				text += '  ' + fun + ': ' + gasToText(data.internal[fun]) + '\n';
-			return text;
-		};
-		var formatAssemblyText = function(asm, prefix, source) {
-			if (typeof(asm) == typeof('') || asm === null || asm === undefined)
-				return prefix + asm + '\n';
-			var text = prefix + '.code\n';
-			$.each(asm['.code'], function(i, item) {
-				var v = item.value === undefined ? '' : item.value;
-				var src = '';
-				if (item.begin !== undefined && item.end != undefined)
-					src = source.slice(item.begin, item.end).replace('\n', '\\n', 'g');
-				if (src.length > 30)
-					src = src.slice(0, 30) + '...';
-				if (item.name != 'tag')
-					text += '  ';
-				text += prefix + item.name + ' ' + v + '\t\t\t' + src +  '\n';
-			});
-			text += prefix + '.data\n';
-			if (asm['.data'])
-				$.each(asm['.data'], function(i, item) {
-					text += '  ' + prefix + '' + i + ':\n';
-					text += formatAssemblyText(item, prefix + '    ', source);
-				});
-
-			return text;
-		};
-		
-		$('.asmOutput button').click(function() {$(this).parent().find('pre').toggle(); });
-
-		var getConstructorInterface = function(abi) {
-			var funABI = {'name':'','inputs':[],'type':'constructor','outputs':[]};
-			for (var i = 0; i < abi.length; i++)
-				if (abi[i].type == 'constructor') {
-					funABI.inputs = abi[i].inputs || [];
-					break;
-				}
-			return funABI;
-		};
-
-	</script>
+    
+
+    <div id="editor">
+        <div id="files">
+            <span class="newFile" title="New File">+</span>
+            <span class="toggleRHP" title="Toggle right hand panel"></span>
+        </div>
+        <div id="input"></div>
+        <div id="dragbar"></div>
+    </div>
+
+    <div id="righthand-panel">
+        <div id="header">
+            <img id="solIcon" src="solidity.svg">
+            <div class="info">
+                <p>
+                    Version: <span id="version">(loading)</span><br/>
+                    Change to: <select id="versionSelector"></select> <button id="gist" title="Share files to a github gist">Share gist</button><br/>
```

Comment:
I think we should be more careful with the wording here, this publishes information which might be confidential or copyrighted.
What about something like `publish as public gist`?